src/jtagmkII.c

```diff
@@ -1215,6 +1219,11 @@ static int jtagmkII_initialize(const PROGRAMMER *pgm, const AVRPART *p) {
   int ok;
   const char *ifname;
 
+  if (PDATA(pgm)->rts_mode != RTS_MODE_DEFAULT) {
+    pmsg_warning("forcing serial DTR/RTS handshake lines %s\n",
```

Comment:
Isn't this a running commentary `pmsg_info()` rather than a warning?

Reply:
I ended up using `pmsg_warning` because that's what the SerialUPDI programmer does. But it would make sense to use `pmsg_info` instead for both the jtag2updi and serialupdi programmer